roles/ceph-mgr/tasks/mgr_modules.yml

```diff
@@ -41,3 +41,13 @@
   with_items: "{{ ceph_mgr_modules }}"
   delegate_to: "{{ groups[mon_group_name][0] }}"
   when: (item in _disabled_ceph_mgr_modules or _disabled_ceph_mgr_modules == [])
+
+- name: check prometheus plugin port
+  command: "{{ container_exec_cmd_mgr | default('') }} ceph --cluster {{ cluster }} config get mgr. mgr/prometheus/server_port"
+  register: ceph_mgr_prometheus_plugin_actual_port
+
+- name: set prometheus plugin port
+  command: "{{ container_exec_cmd_mgr | default('') }} ceph --cluster {{ cluster }} config set mgr mgr/prometheus/server_port {{ ceph_mgr_prometheus_plugin_port }}"
+  delegate_to: "{{ groups[mon_group_name][0] }}"
+  when: ceph_mgr_prometheus_plugin_actual_port.stdout != ceph_mgr_prometheus_plugin_port
```

Comment:
are you sure about this operation ? because it seems that the value returned by the config get is a string but `ceph_mgr_prometheus_plugin_port` is an integer